### DISCUSSION TITLE
Latest release of tfsec 1.28.7 failed to download

### DISCUSSION THREAD

**Discussion by kmcneelyshaw** [aquasecurity/trivy#7049] [Bugs]
### Description

I understand why you want people to migrate from tfsec to trivy, but large institutions cannot turn on a dime.  Unfortunately, without the ability to open issues/bugs/discussions against the tfsec repository, I'm reporting the issue here on a project that might not be supported by the same team.

It seems that recently, source code and metadata about release 1.28.7 were published https://github.com/aquasecurity/tfsec/releases/tag/v1.28.7, but the job to build the binaries failed (I assume it is https://github.com/aquasecurity/tfsec/actions/runs/9703602551 ).  Unfortunately, my CI pipeline said "yay, there is a new version of tfsec! I will go download it!" and then failed to do exactly that.  I can edit my pipelines to statically fetch 1.28.6, but I did want to let you know about the failed build.  Thank you very much for keeping the lights on with tfsec.

### Desired Behavior

When I look up https://github.com/aquasecurity/tfsec/releases/latest and find out 1.28.7 is the latest version published, I will then also be able to resolve https://github.com/aquasecurity/tfsec/releases/download/v1.28.7/tfsec-linux-amd64

### Actual Behavior

No binary was downloaded

### Reproduction Steps

```bash
_desired_file_name="tfsec"
_platform="-linux-amd64"
_org_name="aquasecurity"
_project_name="tfsec"
_latest_version="$(curl -s --head --write-out %{redirect_url} -o /dev/null https://github.com/${_org_name}/${_project_name}/releases/latest | sed 's:.*/::' )"
_file_name=tfsec${_platform}

if [ -z "${TFSEC_VERSION}" ] || [ "${TFSEC_VERSION}" == "latest" ] ; then
  TFSEC_VERSION=${_latest_version}
else
  TFSEC_VERSION="v${TFSEC_VERSION}"
fi

wget -nv -q https://github.com/${_org_name}/${_project_name}/releases/download/${TFSEC_VERSION}/${_file_name}
```

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

### Debug Output

```bash
This is a bug in fetching tfsec, not in running trivy.
```

### Operating System

linux-amd64

### Version

```bash
This is an error in fetching version 1.28.7 of tfsec, not of trivy.
```

### Checklist

- [ ] Run `trivy clean --all`
- [ ] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by simar7** [comment]
Apologies for the breakage, I have a PR to fix that here https://github.com/aquasecurity/tfsec/pull/2148 you can track the progress over there.

**Reply by simar7** [reply]
For the time being I've marked v1.26.0 back as latest as we try to figure out what's going on with docker hub which is preventing us from pushing artifacts.

**Reply by kmcneelyshaw** [reply]
Thank you so much for your responsiveness.

**Reply by simar7** [reply]
@kmcneelyshaw we've fixed the issue with docker hub releases, it should be out now: https://github.com/aquasecurity/tfsec/releases/tag/v1.28.10

Let us know if you have any further questions. Thanks for your patience.

**Reply by kmcneelyshaw** [reply]
Thank you @simar7 .  I've validated that our recent CI runs have picked up 1.28.10 successfully.
